pyproject.toml

```diff
@@ -0,0 +1,16 @@
+[tool.poetry]
+name = "gdcdictionary"
+version = "0.0.0"
+description = "Data Dictionary for MIDRC"
+authors = ["Center for Translational Data Science at the University of Chicago <support@datacommons.io>"]
+license = "Apache-2.0"
+readme = "README.md"
+
+
+[tool.poetry.dependencies]
```

Comment:
since there are no dependencies, it looks like we could even remove `pyproject.toml` and the `poetry install` command from the workflow, and just install `dictionaryutils`

Reply:
will do in ticket [PXP-11255](https://ctds-planx.atlassian.net/browse/PXP-11255)